gettsim/parameters/soz_vers_beitr.yaml

```diff
@@ -10,13 +10,26 @@ beitr_satz:
                     sonderbeitrag - bezahlt von Arbeitnehmern, ab Jul 2005 bis 2014
                     ermäßigt - ermäßigter Satz (§243 SGB V)
                     zusatz - durchschnittlicher Zusatzbeitragssatz.
+      ges_pflegev: Beitragssätze für gesetzliche Pflegeversicherung.
+                    standard - Beitragssatz für Arbeitnehmer
+                    zusatz_kinderlos - Beitragszuschlag für kinderlose Versicherte,
+                    die das 23. Lebensjahr vollendet haben. Wird vom Arbeitnehmer
+                    alleine getragen. Eingeführt 2005.
+      arbeitsl_v:  Beitragssätze zur Arbeitslosenversicherung für Arbeitnehmer.
+      ges_rentenv: Beitragssätze zur allgemeinen Rentenversicherung für Arbeitnehmer.
     en:
-      ges_krankenv: Statury Contribution Rates for mandatory health insurance.
+     ges_krankenv: Statury Contribution Rates for mandatory health insurance.
                     mean_allgemein - average contribution rate, 1992 to 2008
                     allgemein - general statutory contribution rate, since 2009
                     sonderbeitrag - paid by employees, Jul 2005 to 2014
                     ermäßigt - reduced rate
                     zusatz - average top-up contribution rate
+     ges_pflegev: 	Statury Contribution Rates for mandatory care insurance.
```

Comment:
Too many whitespaces before "Statury"

Reply:
Everything should now line up as intended